sp/src/vscript/vscript_squirrel.cpp

```diff
@@ -241,7 +241,9 @@ class SquirrelVM : public IScriptVM
 	HSQOBJECT regexpClass_;
 };
 
-SQUserPointer TYPETAG_VECTOR = "VectorTypeTag";
+// we cast away constness here, but this looks to be an API deficit of squrrel,
+// since the typetag shouldn't be modified
+static SQUserPointer TYPETAG_VECTOR = const_cast<char*>("VectorTypeTag");
```

Comment:
Instead of doing this should be able to just make it a char array

Reply:
Yes, that works.
I didn't know that it's fine to initialize a `char*` from a string literal (while apparently for `void*` it isn't).